index.html

```diff
@@ -0,0 +1,157 @@
+    </p>
+    <h3 class="event-date">2022.11.16(mon) ~ 19(wed)</h3>
+    <h4 class="event-location">@ Al Bayt Stadium | Doha |</h4>
+  </header>
+  <main>
+    <section class="program-section" id="program">
+      <h2 class="sec-title">
+        main program
+        <span class="indicator"></span>
+      </h2>
+      <div class="cards">
+        <div class="card">
+          <img src="icons/microphone.png" alt="speaker-icon" />
+          <h3 id="artiste">Artistes Performing</h3>
+          <p id="musicians">
+            Musicians from different continents will entertain 
+          </p>
+        </div>
+        <div class="card">
+          <img  src="icons/speech.png" alt="speech" /><br>
+          <h3 id="speech" >Speech</h3>
+          <p id="talk">
+            Speech from different football legends of world cup, and from the main speaker(Qatar president)
+          </p>
+        </div>
+        <div class="card">
+          <img src="icons/trophy.png" alt="trophy" />
+          <h3 id="trophy">WorldCup Trophy</h3>
+          <p>
+            The trophy that will be competed for by 32 countries will be unveiled by Qatar President.
+          </p>
+        </div>
+        <div class="card">
+          <img src="icons/sponsors.png" alt="Sponsors" />
+          <h3 id="sponsor">Sponsors</h3>
+          <p id="sponsors">
+            Sponsors for this edition of worldcup will be unveiled by the FIFA president.
+          </p>
+        </div>
+        <div class="card">
+          <img src="icons/tv.png" alt="tv" />
+          <h3 id="tvs">TV broadcast</h3>
+          <p id="tv">
+            All televisions that will broadcast all matches will be unveiled by the FIFA president.
+          </p>
+        </div>
+      </div>
+      <button type="button" class="primary-btn">
+        Don't miss the World's <br> Entertaining Football
+      </button>
+    </section>
+    <section class="speakers-section" id="speakers">
+      <h2 class="dummy-heading">
+        This heading is used for linter pass only
+      </h2>
+      <!-- Generated dynamically -->
+      </section>
+      <section class="partner-section" id="partner">
+        <h2 class="sec-title">
+          partners
+          <span class="indicator"></span>
+        </h2>
+        <div class="partners-logo-container">
+          <img src="images/Partners/cocacola.png" alt="cocacola" />
+          <img src="images/Partners/adidas.png" alt="adidas" />
+          <img src="images/Partners/hp.png" alt="hp-logo" />
+          <img src="images/Partners/MTN-Logo.png" alt="MTN-Logo" />
+          <img src="images/Partners/wenger-logo.png" alt="standard-chartered-logo" />
+
+        </div>
+      </section>
+  </main>
+  <footer>
+    <div class="footer-logo">
+      <img src="images/AMC-logos/Qatar-logo.jpg" id="wc-logo" alt="QWC-logo" />
+    </div>
+
+    <p class="footnote">
+     
+      <span id="copy">&copy; 2022 Qatar World Cup. Some rights reserved. </span>
+    </p>
+  </footer>
```

Comment:
- [ ] The template footer logo is a little bit to the left of the middle of the footer. Please try to follow the template design as much as possible and place the logo to the left as the previous code reviewer suggested. Remember to fix the horizontal alignment too.
 -Your footer:
<img width="1110" alt="image" src="https://user-images.githubusercontent.com/61919617/190453290-f61a59e1-8665-44a6-b4ff-9e1ac747c07c.png">
 -Template footer:
<img width="909" alt="image" src="https://user-images.githubusercontent.com/61919617/190453432-098fa802-b3fa-4038-aae6-f8a4fc572e44.png">
<img width="909" alt="image" src="https://user-images.githubusercontent.com/61919617/190453433-f9e8a68f-aa8b-4c2f-9a0c-ce1c38d657fc.png">

Reply:
Adjust the logo position to the left of the footer